src/components/organisms/AppRouter/AdminSubRouter.tsx

```diff
@@ -10,9 +10,11 @@ import {
   ADMIN_IA_WORLD_CREATE_URL,
   ADMIN_IA_WORLD_EDIT_PARAM_URL,
   ADMIN_IA_WORLD_PARAM_URL,
+  ADMIN_IA_WORLD_TOOLS,
   ADMIN_ROOT_URL,
 } from "settings";
 
+import { Tools } from "pages/Admin/Tools";
```

Comment:
Maybe a good idea to rename it to `ToolsPage`.

Should make maintenance a bit easier if all top level components (directly under a route) end in `*Page`